### ISSUE TITLE
Multiedit of custom JSON fields injects escaping

### ISSUE THREAD

**Issue by paulexyz** [netbox-community/netbox#11749]
### NetBox version

v3.4.4
(newer versionsnot yet tested, but to my knowledge this is not yet fixed)

### Python version

Docker image: netboxcommunity/netbox

### Steps to Reproduce

1. Create a custom field with the type json
2. Select and edit more than one entity having that field
3. populate the field with valid json

### Expected Behavior

Json should be saved to each entities custom field
```json
{
    "a": "foo",
    "b": "bar"
}
```

### Observed Behavior

Json gets escaped and then saved to each enity 
```
"{\n    \"a\": \"foo\",\n    \"b\": \"bar\"\n}"
```

**Comment by kkthxbye-code** [comment]
I cannot replicate this. Please revise your steps to allow other people to replicate it.

**Comment by kkthxbye-code** [comment]
Closing this as there has been no response and I wasn't able to replicate it.

**Comment by paulexyz** [comment]
Sorry it was a bug which was fixed in v3.4.x, I must have tested it in the wrong development instance. Yes I can confirm the bug is not present anymore.
